mimic/rest/identity_api.py

```diff
@@ -669,6 +688,521 @@ def list_tenants(self, request):
                 'message': ("No valid token provided. Please use the 'X-Auth-Token'"
                             " header with a valid token.")}})
 
+    @app.route('/v2.0/services', methods=['GET'])
+    def list_external_api_services(self, request):
+        """
+        List the available external services that endpoint templates
+        may be added to.
+
+        .. note:: Part of the OS-KSADM extension.
+        .. note:: Does not implement the limits or markers.
```

Comment:
Any time you have a reference in a docstring or a comment to desirable-but-unimplemented functionality, or a known bug, it should include a hyperlink referencing the place that the future work is tracked.